gradle.properties

```diff
@@ -124,6 +125,7 @@ artifactRedirecting.androidx.window.version=1.4.0-alpha04
 artifactRedirecting.modules-for-knative-manifest=\
   :annotation:annotation=artifactRedirecting.androidx.annotation.version,\
   :collection:collection=artifactRedirecting.androidx.collection.version,\
+  :graphics:graphics-shapes=artifactRedirecting.androidx.graphics.version,\
```

Comment:
Probably CI fails due to this line:
> No matching variant of androidx.graphics:graphics-shapes:1.0.0 was found.


afair, this is needed only if we actually redirect to androidx-published libs.
For version 1.0.0 there are no k/native artefacts on googleMaven. 
But if we can use 1.0.1, then this line is needed.

Reply:
Updated to 1.0.1